main.py

```diff
@@ -22,7 +23,8 @@
 if args.epub:
     patches.append({'url':'/download/epub/', 'ext':'.epub'})
 
-folder = create_relative_path_if_not_exist('downloads')
+folder = args.folder
+folder = create_path(folder) if folder else create_path('./downloads')
```

Comment:
This one ensures the default download folder is `./download` if user does provide any